multiaccounts/accounts/database.go

```diff
@@ -219,5 +219,5 @@ func (db *Database) AddressExists(address types.Address) (exists bool, err error
 }
 
 func (db *Database) GetNodeConfig() (*params.NodeConfig, error) {
-	return nodecfg.GetNodeConfig(db.db)
+	return nodecfg.GetNodeConfigFromDB(db.db)
```

Comment:
I needed to make this change because `GeNodeConfig` no longer exists. @richard-ramos is that related to you changes?

Reply:
yes, it was surprising that this wasnt caught by the CI